### PR TITLE
Add worker_config with resource `tencentcloud_kubernetes_cluster_attachment`

### PR DESCRIPTION
**Pull request by gailwang**
With existing instances test case, the case need an exsting instance, therefor not put it into test file to make auto-tests.

```
 make testacc TESTARGS=" -run=TestAccTencentCloudTkeAttachExistResource"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./... -v -run=TestAccTencentCloudTkeAttachExistResource -timeout 120m
?       github.com/terraform-providers/terraform-provider-tencentcloud  [no test files]
testing: warning: no tests to run
PASS
ok      github.com/terraform-providers/terraform-provider-tencentcloud/gendoc  (cached) [no tests to run]
=== RUN   TestAccTencentCloudTkeAttachExistResource
--- PASS: TestAccTencentCloudTkeAttachExistResource (336.47s)
PASS
ok      github.com/terraform-providers/terraform-provider-tencentcloud/tencentcloud     337.703s
?       github.com/terraform-providers/terraform-provider-tencentcloud/tencentcloud/connectivity        [no test files]
?       github.com/terraform-providers/terraform-provider-tencentcloud/tencentcloud/internal/helper     [no test files]
?       github.com/terraform-providers/terraform-provider-tencentcloud/tencentcloud/ratelimit   [no test files]

```

### CODE REVIEW

**Comment by gailwang** [comment]
original test
```
make testacc TESTARGS=" -run=TestAccTencentCloudTkeAttachResource"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./... -v -run=TestAccTencentCloudTkeAttachResource -timeout 120m
?       github.com/terraform-providers/terraform-provider-tencentcloud  [no test files]
testing: warning: no tests to run
PASS
ok      github.com/terraform-providers/terraform-provider-tencentcloud/gendoc  (cached) [no tests to run]
=== RUN   TestAccTencentCloudTkeAttachResource
--- PASS: TestAccTencentCloudTkeAttachResource (353.22s)
PASS
ok      github.com/terraform-providers/terraform-provider-tencentcloud/tencentcloud     354.590s
?       github.com/terraform-providers/terraform-provider-tencentcloud/tencentcloud/connectivity        [no test files]
?       github.com/terraform-providers/terraform-provider-tencentcloud/tencentcloud/internal/helper     [no test files]
?       github.com/terraform-providers/terraform-provider-tencentcloud/tencentcloud/ratelimit   [no test files]
```

File: tencentcloud/internal/helper/transform.go [hunk 1]
```diff
@@ -100,3 +100,12 @@ func BoolToInt64Pointer(s bool) (i *uint64) {
 	i = &result
 	return
 }
+
+func BoolToInt64(s bool) (i *int64) {
```
Comment (likexian) [inline tencentcloud/internal/helper/transform.go:104]: BoolToInt64Ptr ?

File: tencentcloud/resource_tc_cam_role.go [hunk 2]
```diff
@@ -25,6 +25,30 @@ EOF
 }
 ```
 
+Created with SAML provider
```
Comment (likexian) [inline tencentcloud/resource_tc_cam_role.go:28]: 这个加 title 了，前面的那个例子也加一下吧

File: tencentcloud/resource_tc_cam_role.go [hunk 3]
```diff
@@ -25,6 +25,30 @@ EOF
...
+
+```hcl
+resource "tencentcloud_cam_role" "boo" {
```
Comment (likexian) [inline tencentcloud/resource_tc_cam_role.go:31]:
```hcl
resource "tencentcloud_cam_role" "boo" {
  name          = "cam-role-test"
  document      = <<EOF
{
  "version": "2.0",
  "statement": [
    {
      "action": ["name/sts:AssumeRole", "name/sts:AssumeRoleWithWebIdentity"],
      "effect": "allow",
      "principal": {
        "federated": ["qcs::cam::uin/<your-account-id>:saml-provider/<your-name>"]
      }
    }
  ]
}
EOF
  description   = "test"
  console_login = true
}
```

File: tencentcloud/resource_tc_kubernetes_cluster_attachment.go [hunk 4]
```diff
@@ -108,6 +108,63 @@ import (
...
+						Description: "Volume of disk in GB. Default is 0.",
+					},
+				},
+			},
+		},
+		"user_data": {
+			Type:        schema.TypeString,
+			ForceNew:    true,
+			Optional:    true,
+			Description: "Ase64-encoded User Data text, the length limit is 16KB.",
```
Comment (likexian) [inline tencentcloud/resource_tc_kubernetes_cluster_attachment.go:156]: Ase64 is what

**Pull request merged**